Automattic/woocommerce-payments
# Disputes List: Renaming `response date` to `respond by`

Fixes #6824 

#### Changes proposed in this Pull Request

Renaming `Response date` column to `Respond by`, values should be hidden when response not required.

Previously, this column showed the original due date of the dispute. Even after the merchant has responded.

This may be confusing for merchants. If the due date is next week then it would say 4 days left however, once the dispute is responded to, it would say July 23, 2023 showing a future date. A merchant would instead expect that the response date is set to the day they responded.

![Screenshot 2023-07-21 at 4 20 16 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/5f2ed2be-fb64-44c0-af7f-e8ae9e08ed28)

Tests were also updated to have a mocked date so snapshots contain the newly update `x days left` in this column from #6702 

#### Testing instructions

1. Visit `Payments > Disputes`
2. Change filter to `All disputes`
3. `Respond by` column should be empty for old disputes.

-------------------


**Post merge**
